### PR TITLE
OSDOCS-4799: Configure OVNH post-install

### PR DESCRIPTION
**Pull request by jboxman-rh**
- https://issues.redhat.com/browse/OSDOCS-4799

Version(s): 4.12+

Issue:

Link to docs preview:
- [Configuring hybrid networking](https://57374--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jboxman-rh** [comment]
@JacobTanenbaum, this PR updates the existing OVHN documentation with steps for configuring post-install. Can you take a look when you have a minute? Thanks!

**Comment by jboxman-rh** [comment]
@anuragthehatter, can you confirm? Thanks!

**Comment by anuragthehatter** [comment]
@jboxman-rh 
```
$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": ["cidr": "10.132.0.0/14", "hostPrefix":23]}}}}}'
Error from server (BadRequest): error decoding patch: invalid character ':' after array element
``` 
complains. Need to check that

Also I believe we need to support null config as well which I believe not working @JacobTanenbaum any thoughts?

```
oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": []}}}}}'
```

File: modules/configuring-hybrid-ovnkubernetes.adoc [hunk 1]
```diff
@@ -87,3 +87,65 @@ Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019 is not su
...
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* Ensure that the cluster uses the OVN-Kubernetes network plugin.
+
+.Procedure
+
+. To configure the OVN-Kubernetes hybrid network overlay, enter the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge \
```
Comment (anuragthehatter) [inline modules/configuring-hybrid-ovnkubernetes.adoc:104]: Correct patch would be 
```
oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": [{"cidr": "10.132.0.0/14", "hostPrefix": 23}]}}}}}'
```

Note ```[{``` before "cidr"

Comment (jboxman-rh) [reply modules/configuring-hybrid-ovnkubernetes.adoc:104]: Oops, array of objects.

**Comment by jboxman-rh** [comment]
> 
> Also I believe we need to support null config as well which I believe not working @JacobTanenbaum any thoughts?
> 
> ```
> oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": []}}}}}'
> ```

@anuragthehatter, `null` would disable the hybrid network if it were implemented?

**Comment by jboxman-rh** [comment]
> 
> Also I believe we need to support null config as well which I believe not working @JacobTanenbaum any thoughts?
> 
> ```
> oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": []}}}}}'
> ```

@anuragthehatter, `null` would disable the hybrid network if it were implemented?

**Comment by anuragthehatter** [comment]
> > Also I believe we need to support null config as well which I believe not working @JacobTanenbaum any thoughts?
> > ```
> > oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": []}}}}}'
> > ```
> 
> @anuragthehatter, `null` would disable the hybrid network if it were implemented?



> > Also I believe we need to support null config as well which I believe not working @JacobTanenbaum any thoughts?
> > ```
> > oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": []}}}}}'
> > ```
> 
> @anuragthehatter, `null` would disable the hybrid network if it were implemented?

Some customers not really using hybrid overlay; but overlay code being reused for something which involves having an F5 router that handles all cluster ingress and egress. SO they really not worry about window containers, so configure cluster with empty array

**Comment by anuragthehatter** [comment]
probbaly have to track this as well meanwhile https://issues.redhat.com/browse/OCPBUGS-10873

**Comment by anuragthehatter** [comment]
This will apply to 4.12+ docs as well I think

**Comment by jboxman-rh** [comment]
@anuragthehatter, sure, I'll include this in 4.12+, but it looks like it doesn't work yet?

**Comment by anuragthehatter** [comment]
> @anuragthehatter, sure, I'll include this in 4.12+, but it looks like it doesn't work yet?

yea, it went in via https://github.com/openshift/cluster-network-operator/pull/1584 in 4.12 but there is one more change awaited from @JacobTanenbaum

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request closed without merging**